presto-docs/src/main/sphinx/release/release-0.217.rst

```diff
@@ -0,0 +1,58 @@
+=============
+Release 0.217
+=============
+
+General Changes
+---------------
+
+* Fix slow query planning performance for queries submitted when cluster did not have the required number of workers available.
+* Fix an issue where a union of a table with a values statement would execute on a single node, which could lead to out of memory errors.
+* Fix partition properties derivation for full outer join.
+* Improve logical plan to remove extra remote exchange before window function when data is already available locally.
+* Add support for grouped execution for window function.
+* Add support for boolean type in EXPLAIN IO.
+* Add support for choosing distribution type for semi join based on cost when ``join_distribution_type='AUTOMATIC'``
+* Add ST_Points function to access vertexes of a linestring.
+* Add configuration property "query.stage-count-warning-threshold".
+* Add ``TOO_MANY_STAGES`` warning when the number of stages exceeds this threshold.
+* Add ``ANALYZE`` statement to the engine which allows collecting table statistics on demand.
+* Add peak user memory usage per task to query stats.
+* Add the amount of data spilled to CLI dialog.
+* Add :doc:`/connector/elasticsearch`.
+* Remove the system memory pool and related configuration properties (``resources.reserved-system-memory``, ``deprecated.legacy-system-pool-enabled``) entirely.
+  System memory pool was deprecated in 0.201, and it was unused by default since that release. All memory allocations will now be served from the general/user memory pool.
+
+
+Web UI
+------
+
+* Improve live plan to show data transfer statistics as edge labels.
+* Improve live plan to contain more information about each node.
+  For example, table scans now show the name of table being scanned.
+* Improve stage skew histograms to make it easier to identify low skew.
+* Add the ability to zoom when viewing plan for completed queries.
+* Add spilled data to query details page.
+
+
+Hive Connector Changes
+----------------------
+
+* Fix ``hive.skip-target-cleanup-on-rollback`` is set, don't delete the target directories when partition creation is rolled back.
+  This fixes an issue where a rollback that only partly succeed could lead to data loss and corrupt Metastore entries.
+* Fix an issue where create table on S3 would fail with an empty S3 prefix.
```

Comment:
Fix an issue where creating a table on S3 could fail for empty S3 "directories"

(I put directories in double quotes as S3 doesn't really have directories, the other way to phrase this is to say "prefixes without associated objects").